### DISCUSSION TITLE
Database update at folder level

### DISCUSSION THREAD

**Discussion by markblinkhorn** [rern/rAudio#1209] [Fixed Issues]
Does 35,000 items in around 3000 folders on  NAS make it a large database? If I select a folder at the start of the alphabet (e.g. Beatles, The) and take the Update Database option then a dialog box with an OK button is shown to confirm my choice and the update is triggered. However, if I select a folder at the end of the alphabet (e.g. Zappa, Frank) then the dialog box does not appear and I cannot start the update. I can however select the upper containing NAS folder and update that - it just takes a long time to complete. I haven't done a binary chop to work out where in the alphabet it trips up.

Thanks
Mark

**Comment by cmh714** [comment]
I dont think so....I have over 116k tracks on 6TB NAS. I havent run into this issue as of yet. GL!!

**Comment by Roland-B** [comment]
@markblinkhorn 

Have you tried increasing the buffer @ Settings > Player > Options > Buffer-Output ?

Had to increase mine to work properly with large music collection, but may not be the issue your having - I currently use "buffer size=65536"

Roland B.

**Reply by markblinkhorn** [reply]
Hi Roland
I had tried doubling it up, but I got bored of trying when I hit 32768. I've just tried 65536 but, unfortunately, with no change.
Thanks for the suggestion though.
Mark

**Reply by Roland-B** [reply]
@markblinkhorn . Apologies, after re-reading what you have said I think you possibly have the same issue I've reported with the GUI and your not doing anything wrong and your folders/music is OK. Also just tested if I can update a folder at bottom of a long list of albums and also then am missing the option to click anything to update. If using windows you can hit enter on the keyboard and it will trigger the confim button.

https://github.com/rern/rAudio-1/discussions/1217

Roland B.

**Reply by markblinkhorn** [reply]
@Roland-B . That makes sense. I can confirm the folder update does happen on pressing enter - it seems the GUI is placing the update dialog box somewhere off the visible window surface and so I concur these 2 issues appear to be related. I'm happy for these reports to be merged and dealt with as one. @rern 

Mark

**Reply by Roland-B** [reply]
@markblinkhorn. I have no idea if can merge someone elses thread, or if can do it how to do so sorry. But appears to be fixed in update [20230202](https://github.com/rern/rAudio-1/releases/tag/20230202), so have closed the bug report I raised. 

Roland.

**Comment by markblinkhorn** [comment]
@Roland-B I can confirm that update [20230202](https://github.com/rern/rAudio-1/releases/tag/20230202) fixes the problem that I reported. I'm not aware of any way of physically merging multiple discussion threads either - however, it seems @rern does review these discussions, and I am appreciative of that.

Mark
